src/js/components/ContainerSettingsComponent.jsx

```diff
@@ -14,11 +14,28 @@ var ContainerSettingsComponent = React.createClass({
   },
 
   getInitialState: function () {
+    var container = this.props.model.container;
+
+    var [portMappingsCount, parametersCount, volumesCount] = Array(3).fill(1);
```

Comment:
Hmm.. I think this is not readable, what is going on. What is wrong with being explicit?
```
var portMappingsCount = 1;
var parametersCount = 1;
var volumesCount = 1;
```